### PR TITLE
Created Rajeshds.json

### PR DESCRIPTION
**Pull request by Rajeshds20**
## Check List (Check all the applicable boxes) 

- [X ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ X] This PR does not contain plagiarized content.
- [ X] The title of my pull request is a short description of the requested changes.

## Screenshots

image
image

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2934"></a>

### CODE REVIEW

File: data/Rajeshds.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "D S Rajesh",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "An enthusiastic Tech Student. I develop softwares and websites, currently working on Full Stack Web Development.",
+  "avatar": "https://avatars.githubusercontent.com/u/104376917?s=400&u=932affed6d49d8afa5a12ebf2cf9b723f3db51eb&v=4",
```
Comment (Aadarsh805) [inline data/Rajeshds.json:6]: use this for the avatar link `https://github.com/Rajeshds20.png`

File: data/Rajeshds.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  "links": [
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/devangamsajjarajesh",
+      "icon": "faLinkedin"
+    }
```
Comment (Aadarsh805) [inline data/Rajeshds.json:12]: missing commas everywhere where each link object

File: data/Rajeshds.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  "name": "D S Rajesh",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "An enthusiastic Tech Student. I develop softwares and websites, currently working on Full Stack Web Development.",
+  "avatar": "https://avatars.githubusercontent.com/u/104376917?s=400&u=932affed6d49d8afa5a12ebf2cf9b723f3db51eb&v=4",
+  "links": [
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/devangamsajjarajesh",
+      "icon": "faLinkedin"
```
Comment (Aadarsh805) [inline data/Rajeshds.json:11]: this icon has a typo it should be `FaLinkedin`

File: data/Rajeshds.json [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      "icon": "FaGithub"
+    }
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/rajeshds55",
+      "icon": "FaTwitter"
+    }
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/dsrajesh20",
```
Comment (Aadarsh805) [inline data/Rajeshds.json:25]: this link is also wrong

File: data/Rajeshds.json [hunk 5]
```diff
@@ -3,38 +3,53 @@
   "displayStatsPublic": true,
   "type": "personal",
   "bio": "An enthusiastic Tech Student. I develop softwares and websites, currently working on Full Stack Web Development.",
-  "avatar": "https://avatars.githubusercontent.com/u/104376917?s=400&u=932affed6d49d8afa5a12ebf2cf9b723f3db51eb&v=4",
+  "avatar": "https://www.github.com/rajeshds20.png",
```
Comment (Aadarsh805) [inline data/Rajeshds.json:6]: the avatar link is still wrong it should be `https://github.com/rajeshds20.png` without the `www`

File: data/Rajeshds.json [hunk 6]
```diff
@@ -3,38 +3,53 @@
...
       "name": "Instagram",
-      "url": "https://www.instagram.com/dsrajesh20",
+      "url": "https://www.instagram.com/rajeshds20",
       "icon": "FaInstagram"
-    }
+    },
+    {
+      "name": "HashNode",
+      "url": "https://rajeshds.hashnode.dev",
+      "icon": "FaHashnode"
```
Comment (Aadarsh805) [inline data/Rajeshds.json:31]: This Hashnode icon doesn't work use this instead `SiHashnode`

File: data/Rajeshds.json [hunk 7]
```diff
@@ -3,38 +3,53 @@
...
+    },
     {
       "name": "Codechef",
       "url": "https://www.codechef.com/users/rajeshds",
       "icon": "FaCodechef"
-    }
+    },
     {
       "name": "Email",
       "url": "dsrajesh71@gmail.com",
```
Comment (Aadarsh805) [inline data/Rajeshds.json:55]: for email use this `FaEnvelope`

File: data/Rajeshds.json [hunk 8]
```diff
@@ -3,38 +3,53 @@
...
     {
       "name": "HackerRank",
       "url": "https://www.hackerrank.com/dsrajesh71",
       "icon": "FaHackerrank"
-    }
+    },
     {
       "name": "Codechef",
       "url": "https://www.codechef.com/users/rajeshds",
       "icon": "FaCodechef"
```
Comment (Aadarsh805) [inline data/Rajeshds.json:51]: for codechef use this `SiCodechef`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/Rajeshds.json [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+      "icon": "FaHackerrank"
+    },
+    {
+      "name": "Codechef",
+      "url": "https://www.codechef.com/users/rajeshds",
+      "icon": "SiCodechef"
+    },
+    {
+      "name": "Email",
+      "url": "dsrajesh71@gmail.com",
```
Comment (Aadarsh805) [inline data/Rajeshds.json:55]: one more thing, This should be a `mailto` link here
change this URL to `mailto:dsrajesh71@gmail.com`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Review by Aadarsh805 — APPROVED** [review]
Alright, everything seems good now
Great Job 👍

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by eddiejaoude** [comment]
Looks good 👍 

Great collaboration 💪 

You can also join us in the EddieHub Discord http://discord.eddiehub.org

**Pull request merged**